src/storage/src/hummock/event_handler/hummock_event_handler.rs

```diff
@@ -578,7 +578,17 @@ impl HummockEventHandler {
                 }
             }
 
-            HummockEvent::LocalSealEpoch { .. } => {}
+            HummockEvent::LocalSealEpoch {
```

Comment:
what means `LocalSealEpoch`? If it only handles the table_watermarks, should we rename it?

Reply:
It means a shard of local state store has called `seal_current_epoch`.